data/tokens.json

```diff
@@ -1296,8 +1300,6 @@
       "value": "{Text.gtmds_color_text-color}",
       "type": "color"
     },
-    "gtmds_accordion_color-disabled": {
-      "value": "{Text.gtmds_color_text-color-disabled}",
```

Comment:
I am OK with removing this style but does that mean we have no such state for the accordion or we indicate disabled by another means e.g. hiding the expand button or something else?

Reply:
Yes, I am not aware of having a disabled state for the accordion and such a need so far, this is why I've removed it.